src/Gui/PreCompiled.h

```diff
@@ -96,8 +96,8 @@
 #include <xercesc/util/TranscodingException.hpp>
 #include <xercesc/util/XMLString.hpp>
 
-#include "InventorAll.h"

```

Comment:
Can you please add a comment because crazy guys like me ;-) might in future like to sort the includes again. Then a note why the order matters would prevent this